src/osal/lv_freertos.c

```diff
@@ -194,10 +194,21 @@ lv_result_t lv_thread_sync_wait(lv_thread_sync_t * pxCond)
     lv_result_t lvRes = LV_RESULT_OK;
 
 #if USE_FREERTOS_TASK_NOTIFY
-    LV_UNUSED(pxCond);
+    TaskHandle_t current_task_handle = xTaskGetCurrentTaskHandle();
+
+    _enter_critical();
+    BaseType_t signal_sent = pxCond->xSignalSent;
```

Comment:
Please use the freertos naming convention: rename `signal_sent` to 'xSignalSent`